### DISCUSSION TITLE
Download is not working

### DISCUSSION THREAD

**Discussion by faizanpmba** [netflie/whatsapp-cloud-api#101] [Q&A]
Following is my partial code but for some reason image is sending just fine but when I try to download the receiving image it just don't work at all and doing nothing. I expected for  it to get downloaded somewhere in my directory and nothing is happening down there. Please let me know how to save the file into the directory?

```
require 'vendor/autoload.php';

use Netflie\WhatsAppCloudApi\Message\Media\LinkID;
use Netflie\WhatsAppCloudApi\Message\Media\MediaObjectID;

$waimgId = $webhookData["entry"][0]["changes"][0]["value"]["messages"][0]["image"]["id"];
$whatsapp_cloud_api->sendTextMessage($senderNumber,  $waimgId);
$media_id = new MediaObjectID($waimgId);
$whatsapp_cloud_api->downloadMedia($media_id);

```

**Comment by aalbarca** [comment]
Hi, see https://github.com/netflie/whatsapp-cloud-api/discussions/103
